pegasus-spark-analyser/src/main/scala/com/xiaomi/infra/pegasus/spark/analyser/PegasusRecord.scala

```diff
@@ -22,20 +21,34 @@ object PegasusRecord {
     )
   }
 
-  private def restoreValue(value: Array[Byte]): Array[Byte] =
-    util.Arrays.copyOfRange(value, 4, value.length)
+  def restoreValue(value: Array[Byte]): Array[Byte]
 
-  def create(rocksIterator: RocksIterator): PegasusRecord = {
-    val keyPair = PegasusRecord.restoreKey(rocksIterator.key)
+  def getPegasusRecord(rocksIterator: RocksIterator): PegasusRecord = {
+    val keyPair = restoreKey(rocksIterator.key)
     new PegasusRecord(
       keyPair.getLeft,
       keyPair.getRight,
-      PegasusRecord.restoreValue(rocksIterator.value)
+      restoreValue(rocksIterator.value)
     )
   }
```

Comment:
You must support something like `restoreExpireTs`, because no matter what data version it is, ttl of the original data can not be missing.

```scala
def restoreExpireTs(value: Array[Byte]) int {
  val bytes = util.Arrays.copyOfRange(value, 0, 4)
  ByteBuffer.wrap(bytes).getInt() 
}
```